### DISCUSSION TITLE
Typo in documentation

### DISCUSSION THREAD

**Discussion by jbemis99** [alan-turing-institute/sktime#5873] [General]
on
https://www.sktime.net/en/stable/examples/AA_datatypes_and_datasets.html#Section-1.1.1:-Time-series---the-%22pd.DataFrame%22-mtype
<quote>
structure convention: obj.index must be monotonous, and one of
</quote>
I believe should be:
must be monotonic

Searching the documentation for monotonous finds 6 occurrences.

**Comment by fkiraly** [comment]
Thanks! Pull request with fix would be appreciated!

Hope your experience with `sktime` isn't too monotonous.

**Comment by jbemis99** [comment]
adding issue https://github.com/sktime/sktime/issues/5878#issue-2115176263
